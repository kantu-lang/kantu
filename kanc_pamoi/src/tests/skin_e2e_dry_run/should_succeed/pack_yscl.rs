use super::*;

#[test]
fn plus_commutative() {
    let path = concat_paths(
        file!(),
        checked_path!(
            "../../sample_code/should_succeed/multi_file/no_warnings/plus_commutative/pack.yscl"
        ),
    );
    let output =
        get_manifest_path_and_backslash_normalized_output(vec![DUMMY_EXEC_PATH, "--pack", &path]);
    insta::assert_debug_snapshot!(output);
}
