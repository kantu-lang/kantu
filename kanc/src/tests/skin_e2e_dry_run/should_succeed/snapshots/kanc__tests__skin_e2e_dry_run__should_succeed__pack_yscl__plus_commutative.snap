---
source: src/tests/skin_e2e_dry_run/should_succeed/pack_yscl.rs
expression: output
---
"Compiled successfully.\nSkipped writing output files, but would have tried writing them to <CARGO_MANIFEST_DIR>/src/tests/sample_code/should_succeed/multi_file/no_warnings/plus_commutative/target.\n"
